Earthfile

```diff
@@ -672,3 +672,31 @@ docs:
     RUN npm run prepare
     RUN HUGO_ENV="production" hugo --gc -b "/local/" -d "public/local"
     SAVE ARTIFACT public /public AS LOCAL docs/public
+
+## ./earthly.sh --push +temp-image --FLAVOR=ubuntu
+## all same flags than the `docker` target plus 
+## - the EXPIRATION time, defaults to 24h
+## - the NAME of the image in ttl.sh, defaults to the branch name + short sha
+## the push flag is optional
+## 
+## you will have access to an image in ttl.sh e.g. ttl.sh/add-earthly-target-to-build-temp-images-339dfc7:24h
+temp-image:
+    FROM alpine 
+    RUN apk add git
+    COPY . ./
+
+    IF [ "$EXPIRATION" = "" ]
+        ARG EXPIRATION="24h"
+    END
+
+    ARG BRANCH=$(git symbolic-ref --short HEAD)
```

Comment:
can we maybe reuse the version target here?

Reply:
good question ... here I was aiming for ease in finding in my local docker images so I can group all the temp images for this branch, or that other branch this also makes it easier to clean up once I know I'm no longer working on that branch ... but I'm open to change it if version is more useful for everyone, or to have a way of choosing one or the other cuz I do agree that it might actually be helpful to have there the flavor and the version as well but then the name might be a bit too long?